app/js/shared/src/main/scala/app/flux/react/app/quiz/QuestionComponent.scala

```diff
@@ -374,6 +374,7 @@ final class QuestionComponent(implicit
       <.div(
         SubComponents.questionTitle(question),
         SubComponents.questionDetail(question),
+        SubComponents.image(question),
```

Comment:
You'll want to put this right above the choicesHolder a couple of lines below (see https://github.com/nymanjens/quizmaster/blob/9afea8bbaee0dd2200d1fcfb5d832d6e908072a9/app/js/shared/src/main/scala/app/flux/react/app/quiz/QuestionComponent.scala#L244)